jenkins/pipelines/cd/tiup/tikv-migration-tiup-mirror-update.groovy

```diff
@@ -0,0 +1,83 @@
+/*
+* @ RELEASE_TAG br-v1.0.0/cdc-v1.0.0/gc-worker-v1.0.0
+* @ STAGING_SERVER false/true
+*/
+
+def tikv_migration_repo_url = "https://github.com/tikv/migration"
+def mirror_server = "http://staging.tiup-server.pingcap.net"
+if (!params.STAGING_SERVER) {
+    mirror_server = "http://prod.tiup-server.pingcap.net"
+}
+
+def download = { os, arch ->
+    sh """
+    name=tikv-\${RELEASE_TAG%-v*}
+    version=v\${RELEASE_TAG#*-v}
+    [ -d package ] || mkdir package
+
+    package_name=\${name}-\${version}-${os}-${arch}.tar.gz
+    download_url=${tikv_migration_repo_url}/releases/download/${RELEASE_TAG}/\${package_name}
+    wget -P package/ \${download_url}
+    """
+}
+
+def publish = { os, arch ->
+    sh """
+    name=tikv-\${RELEASE_TAG%-v*}
+    version=v\${RELEASE_TAG#*-v}
+    desc=""
+    if [ \${name} == "tikv-br" ]; then
+        desc="TiKV cluster backup restore tool"
+    elif [ \${name} == "tikv-cdc" ]; then
+        desc="TiKV-CDC is a change data capture tool for TiKV"
+    elif [ \${name} == "tikv-gc-worker" ]; then
+        desc="GC Worker is a component for TiKV to control the gc process"
+    fi
```

Comment:
Suggest to raise an error or panic in `else` branch.

Reply:
Done